ciao-controller/api/api.go

```diff
@@ -440,12 +440,29 @@ func addWorkload(c *Context, w http.ResponseWriter, r *http.Request) (Response,
 		return errorResponse(err), err
 	}
 
+	// we allow admin to create public workloads for any tenant. However,
+	// users scoped to a particular tenant may only create workloads
+	// for their own tenant.
+	vars := mux.Vars(r)
+	tenantID, ok := vars["tenant"]
+	if ok {
+		req.TenantID = tenantID
+	} else {
+		req.TenantID = "public"
```

Comment:
Ah, I see you comment on it below. Are we opening up any security concerns with this pseudo-tenant "public" - do you also forsee using the same thing for images too..?